content/en/community/hackathons/sessions/baby-steps/content/Setup_Helloworld.md

```diff
@@ -0,0 +1,345 @@
+[    0.000000] Info: [libkvmplat] <bootinfo.c @  101>  000000173000-000000183000 000000010000 rw- 0000000000173000 stck
+[    0.000000] Info: [libkvmplat] <bootinfo.c @  101>  000000183000-000007fe0000 000007e5d000 rw- 0000000000183000
+[    0.000000] Info: [libkvmplat] <bootinfo.c @  101>  000007fe0000-000008000000 000000020000 r-- 0000000007fe0000 rsvd
+[    0.000000] Info: [libkvmplat] <bootinfo.c @  101>  0000fffc0000-000100000000 000000040000 r-- 00000000fffc0000 rsvd
+[    0.000000] Info: [libkvmplat] <bootinfo.c @  101>  00fd00000000-010000000000 000300000000 r-- 000000fd00000000 rsvd
+[    0.000000] Info: [libkvmplat] <setup.c @  374> Switch from bootstrap stack to stack @0x183000
+[    0.000000] Info: [libukboot] <boot.c @  251> Unikraft constructor table at 0x120000 - 0x120018
+[    0.000000] Info: [libukboot] <boot.c @  288> Initialize memory allocator...
+[    0.000000] Info: [libukallocbbuddy] <bbuddy.c @  515> Initialize binary buddy allocator 183000
+[    0.000000] Info: [libukboot] <boot.c @  313> Initialize IRQ subsystem...
+[    0.000000] Info: [libukboot] <boot.c @  320> Initialize platform time...
+[    0.000000] Info: [libkvmplat] <tscclock.c @  255> Calibrating TSC clock against i8254 timer
+[    0.100032] Info: [libkvmplat] <tscclock.c @  276> Clock source: TSC, frequency estimate is 3001267500 Hz
+[    0.100902] Info: [libukboot] <boot.c @  324> Initialize scheduling...
+[    0.101515] Info: [libukschedcoop] <schedcoop.c @  257> Initializing cooperative scheduler
+[    0.103719] Info: [libukboot] <boot.c @  339> Init Table @ 0x120018 - 0x120020
+[    0.104629] Info: [libukbus] <bus.c @  134> Initialize bus handlers...
+[    0.105286] Info: [libukbus] <bus.c @  136> Probe buses...
+[    0.105945] Info: [libkvmpci] <pci_bus_x86.c @  158> PCI 00:00.00 (0600 8086:1237): <no driver>
+[    0.106905] Info: [libkvmpci] <pci_bus_x86.c @  158> PCI 00:01.00 (0600 8086:7000): <no driver>
+[    0.107746] Info: [libkvmpci] <pci_bus_x86.c @  158> PCI 00:02.00 (0300 1234:1111): <no driver>
+[    0.108581] Info: [libkvmpci] <pci_bus_x86.c @  158> PCI 00:03.00 (0200 8086:100e): <no driver>
+Powered by
+o.   .o       _ _               __ _
+Oo   Oo  ___ (_) | __ __  __ _ ' _) :_
+oO   oO ' _ `| | |/ /  _)' _` | |_|  _)
+oOo oOO| | | | |   (| | | (_) |  _) :_
+ OoOoO ._, ._:_:_,\_._,  .__,_:_, \___)
+                  Atlas 0.13.1~5eb820bd
+[    0.112301] Info: [libukboot] <boot.c @  369> Pre-init table at 0x120090 - 0x120090
+[    0.113121] Info: [libukboot] <boot.c @  380> Constructor table at 0x120090 - 0x120090
+[    0.114027] Info: [libukboot] <boot.c @  401> Calling main(1, ['build/helloworld_qemu-x86_64'])
+Hello world!
+Arguments:  "build/helloworld_qemu-x86_64"
+[    0.115738] Info: [libukboot] <boot.c @  410> main returned 0, halting system
+[    0.116550] Info: [libkvmplat] <shutdown.c @   35> Unikraft halted
+```
```

Comment:
Give a brief description of what the debugging messages show (timestamp, debug level, component, source file, line).